### ISSUE TITLE
Issue with  TsLint path

### ISSUE THREAD

**Issue by IGZgustavomarin** [Pablissimo/SonarTsPlugin#55]
I have installed the plugin on SonarQube, and the runner works properly on my machine. When I try to analyze my project all the `.ts` files gets indexed but the TsLint keeps showing this error: 

```
11:15:37.112 DEBUG - Executing TsLint with command: node "/home/guu/myproject/node_modules/tslint/bin/tslint" --format json --config "/home/guu/myproject/tslint.json" "/home/guu/myproject/myapp.ts".
11:15:37.162 ERROR - TsLint Err: module.js:457
11:15:37.162 ERROR - TsLint Err:     throw err;
11:15:37.163 ERROR - TsLint Err:     ^
11:15:37.163 ERROR - TsLint Err:.
11:15:37.163 ERROR - TsLint Err: Error: Cannot find module '/home/guu/myproject/"/home/guu/myproject/node_modules/tslint/bin/tslint"'
11:15:37.163 ERROR - TsLint Err:     at Function.Module._resolveFilename (module.js:455:15)
11:15:37.163 ERROR - TsLint Err:     at Function.Module._load (module.js:403:25)
11:15:37.163 ERROR - TsLint Err:     at Module.runMain (module.js:590:10)
11:15:37.163 ERROR - TsLint Err:     at run (bootstrap_node.js:394:7)
11:15:37.163 ERROR - TsLint Err:     at startup (bootstrap_node.js:149:9)
11:15:37.163 ERROR - TsLint Err:     at bootstrap_node.js:509:3
11:15:37.164 DEBUG - Executing TsLint with command: node "/home/guu/myproject/node_modules/tslint/bin/tslint" --format json --config "/home/guu/myproject/tslint.json" "/home/guu/myproject/test/testUtils.ts"
11:15:37.213 ERROR - TsLint Err: module.js:457
11:15:37.214 ERROR - TsLint Err:     throw err;
11:15:37.214 ERROR - TsLint Err:     ^
11:15:37.214 ERROR - TsLint Err:.
11:15:37.214 ERROR - TsLint Err: Error: Cannot find module '/home/guu/myproject/"/home/guu/myproject/node_modules/tslint/bin/tslint"'
11:15:37.214 ERROR - TsLint Err:     at Function.Module._resolveFilename (module.js:455:15)
11:15:37.214 ERROR - TsLint Err:     at Function.Module._load (module.js:403:25)
11:15:37.214 ERROR - TsLint Err:     at Module.runMain (module.js:590:10)
11:15:37.214 ERROR - TsLint Err:     at run (bootstrap_node.js:394:7)
11:15:37.214 ERROR - TsLint Err:     at startup (bootstrap_node.js:149:9)
11:15:37.214 ERROR - TsLint Err:     at bootstrap_node.js:509:3
11:15:37.252 INFO  - Sensor com.pablissimo.sonar.TsLintSensor (done) | time=215ms
```

If I copy&paste the command and run it, it works fine. 

```
node "/home/guu/myproject/node_modules/tslint/bin/tslint" --format json --config "/home/guu/myproject/tslint.json" "/home/guu/myproject/myapp.ts".
```

What may be wrong when executing through `sonar-runner`? Any ideas?

**Comment by Pablissimo** [comment]
Same issue as #47, can you try https://github.com/Pablissimo/SonarTsPlugin/files/440852/sonar-typescript-plugin-0.92-SNAPSHOT.zip?

**Comment by Pablissimo** [comment]
Closed with v0.94, now released.
